main/editor.js

```diff
@@ -17,11 +17,26 @@ const url = isDev ? devPath : prodPath;
 
 let editor = null;
 
-const openEditorWindow = () => {
+const openEditorWindow = ({alwaysOnTop = false} = {}) => {
   if (!editor) {
+    const width = 768;
+    const barHeight = 48;
+    const height = (768 * (9 / 16)) + barHeight;
```

Comment:
It's not clear what the other numbers here are for. Can you move all the numbers out into descriptive variables?